components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/OIDCSessionConstants.java

```diff
@@ -31,6 +31,7 @@
     public static final String OIDC_POST_LOGOUT_REDIRECT_URI_PARAM = "post_logout_redirect_uri";
     public static final String OIDC_STATE_PARAM = "state";
     public static final String OIDC_SESSION_DATA_KEY_PARAM = "sessionDataKey";
+    public static final String TENANT_DOMAIN = "tenantDomain";
```

Comment:
tenantDomain is abstract, shouldn't we specifically say which tenantDomain we are looking for?

Reply:
This is the name of the request parameter. Same query param name is used with token endpoint and authz endpoint to specify user tenant domain. Therefore I used the same here for consistency.
[1] https://docs.wso2.com/display/IS570/Try+Authorization+Code+Grant

Reply:
Actually, authorize URL doesn't make use of this tenantDomain parameter. Internally tenantDomin is derived based on the client_id. 
We need to fix the docs I guess.